kfs-purap/src/main/resources/edu/arizona/kfs/workflow/RequisitionDocument.xml

```diff
@@ -0,0 +1,101 @@
+          <split name="HasAccountingLines" nextNode="SubAccount">
+            <branch name="True">
+              <simple name="NoOpHasAccountingLines" nextNode="JoinHasAccountingLines" />
+            </branch>
+            <branch name="False">
+              <role name="Initiator" nextNode="JoinHasAccountingLines" />
+            </branch>
+            <join name="JoinHasAccountingLines" />
+          </split>
+          <role name="SubAccount" nextNode="Account" /> 
+          <role name="Account" nextNode="AccountingOrganizationHierarchy" />
+          <role name="AccountingOrganizationHierarchy" nextNode="ObjectSubTypeCode" />
+          <role name="ObjectSubTypeCode" nextNode="Commodity" />
+          <role name="Commodity" nextNode="AmountRequiresSeparationOfDutiesReview" />
+          <split name="AmountRequiresSeparationOfDutiesReview">
+            <branch name="True">
+              <role name="SeparationOfDuties" nextNode="Join" />
+            </branch>
+            <branch name="False">
+              <simple name="NoOp" nextNode="Join" />
+            </branch>
+            <join name="Join" />
+          </split>
+        </routePath>
+      </routePaths>
+      <routeNodes>
+        <start name="AdHoc" />
+        <role name="Organization">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <split name="HasAccountingLines">
+          <type>org.kuali.kfs.sys.document.workflow.SimpleBooleanSplitNode</type>
+        </split>
+        <simple name="NoOpHasAccountingLines">
+          <type>org.kuali.rice.kew.engine.node.NoOpNode</type>
+        </simple>
+        <join name="JoinHasAccountingLines" />
+        <role name="Initiator">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <role name="SubAccount">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <role name="Account">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+          <mandatoryRoute>true</mandatoryRoute>
+        </role>
+        <role name="AccountingOrganizationHierarchy">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <role name="ObjectSubTypeCode">
+          <qualifierResolverClass>org.kuali.rice.kns.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
```

Comment:
should be:
org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver